spec/classes/dhcp_spec.rb

```diff
@@ -22,6 +22,7 @@
         osfamily: 'RedHat',
         operatingsystem: 'RedHat',
         operatingsystemrelease: '6',
+        operatingsystemmajrelease: '6',
```

Comment:
I can do, but there's no test at the moment for `/etc/sysconfig/dhcpd` either.  Should I add one?